content/en/docs/Best practices/site-guidance.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Guidance and tips for authoring in Docsy"
+linkTitle: "Docsy guidance and tips"
+weight: 9
+description: >
+  Optional guidance about authoring Docsy and Hugo based website content.
+---
+
+As mentioned in the [About](/about') page, Docsy is a pre-configured theme for the [Hugo](https://gohugo.io/) static site 
+generator. If you're not already familiar with working with Hugo and, in particular, its version of Markdown, this page provides some useful tips and potential gotchas for adding and editing content for your site.
+
+### Nested lists
+
+Hugo currently uses the [Blackfriday](https://github.com/russross/blackfriday) Markdown processor, which can be 
+sensitive when it come to content that's deeply nested in a list. In particular be aware that
+[this known issue](https://github.com/russross/blackfriday/issues/329) can surface if or when you have multiple authors and
+other contributors who might mix 'tabs' and 'spaces' when indenting lists, or fail to indent properly.
+
+An additional factor here is that because GitHub uses a different Markdown processor, GitHub markdown and the editor UI may
+render some nested lists as expected, while Blackfriday might render that same content poorly. For example, the count in a
+numbered list might restart, or your nested content within a list is not indented 
+(shows as a peer element instead of a nested child element). You may want to recommend in your contribution guidelines
+([as we do](/docs/contribution-guidelines/#contributing-to-these-docs)) that contributors preview their content with Hugo
+(or use Netlify's preview feature for PRs if that's your chosen deployment tool) to ensure their content renders correctly
+with Blackfriday.
+
+{{% alert title="Tip" %}}
+[Per comments on the known issue](https://github.com/russross/blackfriday/issues/329#issuecomment-277602856), some
+users have found that using 4 spaces instead of a 'tab' results in consistent behavior. For example, consider
+configuring your local editor to use 4 spaces when the **Tab** key is pressed.
+{{% /alert %}}
+
+### Linking
```

Comment:
I didn't edit this section as I wasn't entirely sure what you were getting at here - the specific issue I think is that all relative URLs (if you specify links  the usual Markdown way) end up being rendered relative to the site root, which might not work in eg the Github UI. I've just seen that there's a Hugo feature to make URLs relative to the current content rather than absolute, I am going to have an experiment.... https://gohugo.io/content-management/urls/#relative-urls

Reply:
Yes please try too. I never got to the source of our problems. Our content was full of relative links, some moving up two or three directories and then back down into other sub directories, and most of them resulted in 404s. I think part of my challenge was the use of the "readfile" shortcode, which also behaved poorly with relative links. Ultimately, this was a complete surprise to our team and I want to avoid that surprise for others. Regarding the relative-urls, there are posts in the forum that suggest you need to turn on both canonifyURLs = "true" and relativeURLs = "true" (but neither fix our issues).

Reply:
Updated!